db/migrations/0009.add_meta_tables.py

```diff
@@ -0,0 +1,33 @@
+#
+# file: migrations/0009.add_meta_tables.py
+#
+from yoyo import step
+
+__depends__ = {'0008.canvas_usage_table'}
+
+step('''
+    CREATE TABLE IF NOT EXISTS job_run
+    (
+        id INTEGER NOT NULL UNIQUE AUTO_INCREMENT,
+        job_name VARCHAR(50) NOT NULL,
+        started_at DATETIME NOT NULL,
```

Comment:
@pushyamig went with `DATETIME` earlier, so I've just gone with it. I feel like maybe there could be less confusion if we just always write UTC timestamps to the database, rather than allowing MySQL to handle that part of the process. Let's discuss later.